### PR TITLE
Add some dependencies to build by exrm

### PR DESCRIPTION
**Pull request by xtity**
I found a problem to build [in this way](http://www.phoenixframework.org/v0.10.0/docs/advanced-deployment).
I think it's up to lack of some dependencies to build by exrm.

So, I added some dependencies. Please check out my code.

### CODE REVIEW

**Comment by chrismccord** [comment]
Phoenix shouldn't depend on exrm, but we should add :ecto to the applications list if ecto is enabled in the phoenix.new task. Can you update the PR with those changes?

**Comment by josevalim** [comment]
Also, we should not include ecto if the user is skipping it. Take a look at deps in the installer template mix.exs.

**Comment by xtity** [comment]
@chrismccord , @josevalim OK. I'm going to update my PR. Thank you for your quick response.

**Comment by xtity** [comment]
Rebased on ``` master ```

File: mix.exs [hunk 1]
```diff
@@ -32,6 +32,7 @@ defmodule Phoenix.Mixfile do
      {:poison, "~> 1.3"},
      {:redo, github: "heroku/redo", optional: true},
      {:poolboy, "~> 1.4.2", optional: true},
+     {:ecto, "~> 0.1", optional: true},
```
Comment (josevalim) [inline mix.exs]: This still should not be here. Can you please revert the changes to `Phoenix` mixfile? THank you!

**Comment by xtity** [comment]
@josevalim I reverted those changes and rebased on master again.

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:

**Comment by josevalim** [comment]
:heart: :green_heart: :blue_heart: :yellow_heart: :purple_heart:
